src/classes/PMT_PaymentWizard_CTRL.cls

```diff
@@ -238,7 +238,9 @@ public with sharing class PMT_PaymentWizard_CTRL {
             writeoffPaymentCount = outstandingPaymentCount;
         }
 
-        
+        if (invalidPaymentRecordCount > 0) {
+            isWizardEnabled = false;
+        }
     }
```

Comment:
@seanpat09 - sorry - but last minute bug -that I found.  There should be a check here for invalid payments on the schedule payment wizard page.  Change is to add a check and then set the wizard to be false.

Reply:
No worries, nice work catching that!